### ISSUE TITLE
Running jco produced js component from example host fails

### ISSUE THREAD

**Issue by bobajeff** [bytecodealliance/component-docs#162]
Running the js component, created following [this section](https://component-model.bytecodealliance.org/language-support/javascript.html) of the guide, from the [example host](https://github.com/bytecodealliance/component-docs/tree/main/component-model/examples/example-host)  no longer works. I guess because it's missing some wasi bits.

```sh
Error: Failed to instantiate the example world

Caused by:
    0: import `wasi:http/types@0.2.0` has the wrong type
    1: instance export `fields` has the wrong type
    2: expected resource found nothing
```

_Note transpiling and running from node.js works still._

**Comment by vados-cosmonic** [comment]
**tl;dr - add `--disable all` to your `jco componentize` call**

Hey @bobajeff while we've been updating the JS guides recently (they'll look very different, very soon), with the currently published site as-is I *was* able to reproduce the issues you ran into with the Rust host and more recent versions of `jco`.

Other than the path (`component-model/examples/add-host`) being wrong, running with the host *does* fail errors about `wasi` interfaces being missing (I get a different set). To explain it succinctly, this is because WASI capabilities aren't actually loaded into the host code in Rust (no need, of course, since in the examples all we're doing is adding!).

The fix to this is to use `--disable all` when running `jco componentize` -- WASI capabilities are built in by default (as that is the more common use case moving forward), but the Rust host is quite minimal and doesn't build in those capabilities. The `--disable` option instructs `jco` not to build in/expect those capabilities to be present at runtime.

Also, while you're poking around, you might want to update your dependencies to the latest `jco` and `componentize-js` versions ([0.7.1](https://www.npmjs.com/package/@bytecodealliance/jco/v/1.7.1) and [0.13.1](https://www.npmjs.com/package/@bytecodealliance/componentize-js/v/0.13.1) respectively right now) :)

**Comment by bobajeff** [comment]
Thanks! `--disable all` does the trick. I look forward to seeing the updated js guides.
